tools/wpt/update.py

```diff
@@ -5,28 +5,52 @@
 sys.path.insert(0, os.path.abspath(os.path.join(wpt_root, "tools")))
 
 
+def manifest_update(test_paths):
+    from manifest import manifest
+    for url_base, paths in test_paths.items():
+        manifest_path = paths["manifest_path"]
+        manifest.load_and_update(
+            paths["tests_path"],
+            manifest_path,
+            url_base)
```

Comment:
Any particular reason only one bit is extracted into a variable? Simpler this way? 

```suggestion
        manifest.load_and_update(
            paths["tests_path"],
            paths["manifest_path"],
            url_base)
```